server/controllers/index.js

```diff
@@ -63,13 +63,15 @@ router.get('/api/start', auth, (req, res) => {
     res.sendStatus(200);
 });
 
-/* Check if player is logged in */
-router.get('/api/end', auth, (req, res) => {
-    currentUsers = []
-    gameLogic.endGame();
-    req.session.destroy();
-    res.sendStatus(200);
-});
+/* Check if in dev mode, and enable end game request */
```

Comment:
I think there are cases where NODE_ENV can be different things e.g. development, production, etc. so possibly change this to `if (process.env.NODE_ENV === 'testing')` and then set this environment variable when we start the server by doing `npx cross-env NODE_ENV='testing' nodemon server.js`? Also could the api call be more descriptive like 'reset server' or something?

Reply:
agreed -- the content is important, not just the truthiness. Tilly's code looks correct.

Reply:
Thanks! Adding it now...